apps/mobile/src/screens/Login/LandingScreen.tsx

```diff
@@ -16,10 +20,16 @@ import { EmailIcon } from './EmailIcon';
 import { LandingLogo } from './LandingLogo';
 import { QRCodeIcon } from './QRCodeIcon';
 
+const IS_ONBOARDING_ENABLED = true;
```

Comment:
Seems like we can't used feature flag since the user didn't logged in yet, so we can't check the role. 

Replaced it with hardcode data. Lmk if there is any alternative.